lib/pages/chat_list/chat_list_body_view.dart

```diff
@@ -48,7 +49,9 @@ class ChatListBodyView extends StatelessWidget {
           ),
           stream: controller.activeClient.onSync.stream
               .where((s) => s.hasRoomUpdate)
-              .rateLimit(const Duration(seconds: 1)),
+              .doOnData((event) async {
+            await controller.updateClientNameBox(controller.activeClient);
```

Comment:
We do on `onData`, but we dont care about data. Why?

Reply:
This is the first time, client get all rooms, i will think more in afternoon

Reply:
maybe you miss a lot of case related to roomUpdate, member in, member out, ban, kick, ....